test/core/test_operator.py

```diff
@@ -261,6 +261,50 @@ def test_run_job_not_concurrent(self):
         assignment = task_run.get_assignments()[0]
         self.assertEqual(assignment.get_status(), AssignmentState.COMPLETED)
 
+    @patch("mephisto.operations.operator.RUN_STATUS_POLL_TIME", 1.5)
+    def test_patience_shutdown(self):
+        """Ensure that a job can be run that doesn't require connected workers"""
```

Comment:
Not sure id this is the right docstring for this test function.

Reply:
Good catch!